tests/wasm_worker/shared_memory.c

```diff
@@ -0,0 +1,6 @@
+#include <emscripten.h>
+
+int main()
+{
+	_ReportResult(EM_ASM_INT(return buffer instanceof SharedArrayBuffer), 0);
```

Comment:
In general, these days you should be able to just use the normal `assert` + `return 0` in all the browser tests.  And run them via `btest_exit` rather than `btest`.  This makes the tests run both in node and in the browser, make them look more like normal tests.   Also the result failed asserts give a line number which makes debugging easier when we see failures on the bots.

Not that its relevant here since this test doesn't need to be a browser test anyway.

Reply:
The test wants to return either 0 or 1, it tests things both ways. `btest_exit` only works for `return 0`

Reply:
`best_exit` takes expected=N for the returncode if you want to return different things (it just less common because the default of zero is often what is desired).

Reply:
Migrated to test_other.py